### ISSUE TITLE
About loss issue

### ISSUE THREAD

**Issue by laoniandisko** [foamliu/Speech-Transformer#17]
I trained your source code, and after training for 10epochs, the loss basically does not change, it is still very large, and the test results are very bad. I would like to know if your pretrained model has any structure to change the code

**Comment by vivektyagiibm** [comment]
I agree with @laoniandisko  observation and I also ran into the same problem. The loss gets stuck and the recognition results are very poor. There seems to be a bug in training code.
